@xen-orchestra/web-core/lib/components/ui/alarm-item/UiAlarmItem.vue

```diff
@@ -0,0 +1,144 @@
+<!-- v1 -->
+<template>
+  <li class="ui-alarm-item" :class="className">
+    <div class="content">
+      <div class="label-value text-ellipsis">
```

Comment:
```suggestion
      <div class="label-value text-ellipsis">
```

Maybe find another name for the class, and since we use `text-ellispis` you should add a `v-tooltip`